src/GitVersion.Core.Tests/VersionCalculation/Strategies/MergeMessageBaseVersionStrategyTests.cs

```diff
@@ -65,7 +66,7 @@ public void TakesVersionFromMergeOfReleaseBranch(string message, bool isMergeCom
         AssertMergeMessage(message + "\n ", expectedVersion, parents);
     }
 
-    [TestCase("Merge branch 'hotfix-0.1.5'", false)]
+    // [TestCase("Merge branch 'hotfix-0.1.5'", false)] this is somehow configured as a release branch
```

Comment:
Can you please elaborate on how `hotfix-0.1.5` is configured as a release branch? Is `is-release-branch` set to `true`?

Reply:
Yes, the `is-release-branch` is set to `true`.
Here's an excerpt of the generated configuration:

``` yaml
hotfix:
      mode: ManualDeployment
      label: beta
      increment: Inherit
      prevent-increment:
        when-current-commit-tagged: false
      regex: ^hotfix(es)?[/-](?<BranchName>.+)
      source-branches:
      - release
      - main
      - support
      - hotfix
      is-source-branch-for: []
      is-release-branch: true
      pre-release-weight: 30000
```

Unfortunately I can't tell whether this is on purpose or not.